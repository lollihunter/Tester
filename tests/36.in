100 100 100
1 1
63 1
22 37
22 97
74 27
62 50
86 50
88 46
94 17
6 65
40 66
15 60
31 52
23 96
36 7
13 29
6 40
90 94
21 34
84 47
83 98
85 89
8 66
35 4
25 20
62 67
93 6
85 21
11 75
13 8
14 87
87 42
7 39
40 93
1 94
31 20
76 29
28 28
8 34
89 37
13 89
38 1
100 40
82 59
61 59
56 58
69 21
88 17
34 46
96 99
65 83
55 39
36 24
54 70
26 95
80 86
40 27
35 71
18 43
38 34
55 22
77 81
87 81
78 93
83 40
15 19
74 78
80 94
42 91
91 99
89 80
13 39
100 33
8 10
75 4
100 57
63 75
96 73
68 4
56 47
26 56
63 13
40 1
18 31
16 38
75 63
2 61
48 67
96 33
54 61
47 75
22 53
60 57
84 6
62 61
77 83
83 84
23 65
10 88
100 100
