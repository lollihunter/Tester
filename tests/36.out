30
1 74 66 16 39 72 59 92 12 98 58 11 88 90 46 93 45 95 26 77 51 67 62 96 56 22 18 70 50 100 
